Traceback (most recent call last):
  File "/homes/rydzipx/good_flat_testing_script/aap_prefix2.py", line 95, in <module>
    process_yaml_file(filename, prefix, keys, lookup_table)
  File "/homes/rydzipx/good_flat_testing_script/aap_prefix2.py", line 53, in process_yaml_file
    original_data = yaml.safe_load(f)
  File "/usr/lib64/python3.9/site-packages/yaml/__init__.py", line 162, in safe_load
    return load(stream, SafeLoader)
  File "/usr/lib64/python3.9/site-packages/yaml/__init__.py", line 114, in load
    return loader.get_single_data()
  File "/usr/lib64/python3.9/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
  File "/usr/lib64/python3.9/site-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/usr/lib64/python3.9/site-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/usr/lib64/python3.9/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/usr/lib64/python3.9/site-packages/yaml/composer.py", line 133, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
  File "/usr/lib64/python3.9/site-packages/yaml/composer.py", line 82, in compose_node
    node = self.compose_sequence_node(anchor)
  File "/usr/lib64/python3.9/site-packages/yaml/composer.py", line 111, in compose_sequence_node
    node.value.append(self.compose_node(node, index))
  File "/usr/lib64/python3.9/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/usr/lib64/python3.9/site-packages/yaml/composer.py", line 133, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
  File "/usr/lib64/python3.9/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/usr/lib64/python3.9/site-packages/yaml/composer.py", line 133, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
  File "/usr/lib64/python3.9/site-packages/yaml/composer.py", line 64, in compose_node
    if self.check_event(AliasEvent):
  File "/usr/lib64/python3.9/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/usr/lib64/python3.9/site-packages/yaml/parser.py", line 449, in parse_block_mapping_value
    if not self.check_token(KeyToken, ValueToken, BlockEndToken):
  File "/usr/lib64/python3.9/site-packages/yaml/scanner.py", line 116, in check_token
    self.fetch_more_tokens()
  File "/usr/lib64/python3.9/site-packages/yaml/scanner.py", line 239, in fetch_more_tokens
    return self.fetch_literal()
  File "/usr/lib64/python3.9/site-packages/yaml/scanner.py", line 635, in fetch_literal
    self.fetch_block_scalar(style='|')
  File "/usr/lib64/python3.9/site-packages/yaml/scanner.py", line 649, in fetch_block_scalar
    self.tokens.append(self.scan_block_scalar(style))
  File "/usr/lib64/python3.9/site-packages/yaml/scanner.py", line 989, in scan_block_scalar
    chomping, increment = self.scan_block_scalar_indicators(start_mark)
  File "/usr/lib64/python3.9/site-packages/yaml/scanner.py", line 1087, in scan_block_scalar_indicators
    raise ScannerError("while scanning a block scalar", start_mark,
yaml.scanner.ScannerError: while scanning a block scalar
  in "job_templates.yaml", line 15558, column 15
expected chomping or indentation indicators, but found '2'
  in "job_templates.yaml", line 15558, column 17
